autospec/pkg_integrity.py

```diff
@@ -940,7 +940,7 @@ def load_specfile(specfile):
 
 def main(args):
     """Verify tar content with signature."""
-    from_disk(args.url, args.tar, args.sig, config.Config())
+    from_disk(args.url, args.tar, args.sig, config.Config(""))
```

Comment:
Hmm, I think I would prefer removing the `main` function entirely at this point, since all references to `config.download_path` in the rest of the file will yield unexpected results.

Reply:
It was a handy idea but we really would need to improve configuration handling to bring it back so dropping it seems like the right move at this point.